src/PanelTraits/Create.php

```diff
@@ -120,4 +136,86 @@ public function syncPivot($model, $data, $form = 'create')
             }
         }
     }
+
+    private function createOneToOneRelations($item, $data, $form = 'create')
+    {
+        $relationData = $this->formatData($data, $form);
+        $this->createRelationsForItem($item, $relationData);
+    }
+
+    private function createRelationsForItem($item, $formattedData)
+    {
+        foreach ($formattedData['relations'] as $relationMethod => $relationData) {
+            $parent = $item;
```

Comment:
why do you need to store it in a temporary variable if you are not going to use `$item` again?